From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew Branson <andrew.branson@jolla.com>
Date: Wed, 7 Jul 2021 14:33:37 +0200
Subject: [PATCH] Completely disable doc generation because we don't have
 xsltproc either.

---
 meson.build | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 595777832eb9d54bf3f835da45a970682ae3233a..f780d3f3f0b24615d2d3cdb01b854b488b713e99 100644
--- a/meson.build
+++ b/meson.build
@@ -200,7 +200,9 @@ darwin_versions = [current, '@0@.@1@'.format(current, interface_age)]
 subdir('src')
 subdir('po')
 subdir('data')
-subdir('doc')
+if get_option('api-docs')
+  subdir('doc')
+endif
 
 if get_option('examples')
     subdir('examples')
