From 3eda08d6fb1359e6d7886cb8941282205c9f4efe Wed, 7 Jul 2021 14:34:05 +0200
From: Andrew Branson <andrew.branson@jolla.com>
Date: Wed, 7 Jul 2021 14:33:37 +0200
Subject: [PATCH] Completely disable doc generation because we don't have xsltproc either.


diff --git a/meson.build b/meson.build
index 79578ee..8a298cb 100644
--- a/meson.build
+++ b/meson.build
@@ -185,7 +185,9 @@
 subdir('src')
 subdir('po')
 subdir('data')
-subdir('doc')
+if get_option('api-docs')
+  subdir('doc')
+endif
 
 if get_option('examples')
     subdir('examples')
